process.md

```diff
@@ -61,126 +62,331 @@ This is for two main reasons:
+Process of becoming an Maintainer:
 
-To be added as an OWNER of most repositories, you must:
+1. Any current Maintainer may nominate a current Collaborator to become a new
+   Maintainer, by opening a PR against the appropriate project/directory
+   [OWNERS file](https://www.kubernetes.dev/docs/guide/owners/), adding their
+   GitHub username to the `approvers` list (or corresponding
+   [OWNERS alias](https://www.kubernetes.dev/docs/guide/owners/#owners_aliases)).
+2. The nominee will add a comment to the PR testifying that they agree to all
+   requirements of becoming a Maintainer.
+3. A majority of the current Maintainers must then approve the PR.
+4. Update [org.yaml](./org/org.yaml) to add the new maintainer to the
+   corresponding
+   [GitHub team(s)](https://docs.github.com/en/organizations/organizing-members-into-teams/about-teams).
 
-* Have been actively participating in reviews for at least 3 months or
-  50% of the project lifetime, whichever is shorter
-* Have been the primary reviewer for at least 10 substantial PRs to the codebase.
-* Have reviewed at least 30 PRs to the codebase.
-* Be nominated by another OWNER (with no objections from other OWNERS)
+- Each project has a `<repo>.maintainers` entry in [`org.yaml`](./org/org.yaml),
+  where `<repo>` is the name of the GitHub repository. The only exception is
+  `pipeline` whose maintainer team is name `core.maintainers`.
 
-The final change will be made via a PR to update the OWNERS file.
+### Governance Committee Member
 
-To facilitate productivity, small repositories, or repositories that do not contain production
-code may decide to use a simpler OWNERs process.
-To become an OWNER of one of these repositories, you must either:
+Description: The Tekton Governance committee is the governing body of the Tekton
+open source project. It's an elected group that represents the contributors to
+the project, and has an oversight on governance and technical matters.
 
-* Be an OWNER on any other repository in the Tekton project, and ask an existing OWNER to add you.
-* Or, Be nominated by another OWNER (with no objections from other OWNERs)
+See [governance.md](governance.md) for requirements, responsibilities, and
+election process.
 
-Repositories currently using this simpler mechanism are:
+- Additional privileges:
+  - Maintainer privileges on all Tekton projects
+  - Organization admin access.
 
-* tektoncd/community
-* tektoncd/friends
-* tektoncd/plumbing
-* tektoncd/results
-* tektoncd/website
-* tektoncd/experimental
+## Inactivity
 
-#### Requirements for Collaborators
+It is important for contributors to be and stay active to set an example and
+show commitment to the project. Inactivity is harmful to the project as it may
+lead to unexpected delays, contributor attrition, and a lost of trust in the
+project.
 
-To be added as collaborator of most repositories, you must:
+- Inactivity is measured by:
+  - Periods of no contributions for longer than 3 months
```

Comment:
probably good to define "contribution" here - i think this means not meeting the requirements for the particular role?

Reply:
Added a contributions section above based on the voter eligibility requirements.

I think we want to keep this as contributions instead of requirements though. - we define requirements pretty broadly and on a yearly basis to give people leeway for contributions over time. This is basically a check to ensure that participation is happening on a regular basis rather than a burst a short period of time.

e.g. for collaborator we define a requirement as 10 PRs a year. Ideally ~1 PR / month would be better than 10 PRs in one month, then little/no engagement for the rest of the year.

Thinking over this some more made me realize that no communication would almost certainly happen before no-contributions, so tweak the numbers a bit.

Let me know what you think!